packages/pigeon/lib/cpp_generator.dart

```diff
@@ -605,14 +605,10 @@ const flutter::StandardMessageCodec& ${api.name}::GetCodec() {
                 final String errorGetter;
                 final String prefix = (reply != '') ? '\t' : '';
 
-                const String resultKey =
-                    'flutter::EncodableValue("${Keys.result}")';
-                const String errorKey =
-                    'flutter::EncodableValue("${Keys.error}")';
                 const String nullValue = 'flutter::EncodableValue()';
                 if (returnType.isVoid) {
                   elseBody =
-                      '$prefix\twrapped.emplace($resultKey, $nullValue);${indent.newline}';
+                      '$prefix\twrapped.push_back($nullValue);${indent.newline}';
```

Comment:
I wrote the `push_back` here; see https://abseil.io/tips/112

I'm skeptical that there's actually a perf difference since `push_back` of an a rvalue is almost certainly a move.

Reply:
Yea, I'm not sure.  I'd have to look to see what `flutter::EncodableValue` is.  I'm fine with `push_back`.  If we add a benchmark for c++ would be interesting to try.  Since it is our class we probably aren't in danger of it accidentally grabbing an implicit constructor.